lib/discourse_chat/provider/slack/slack_provider.rb

```diff
@@ -40,7 +40,7 @@ def self.slack_message(post, channel)
 
     message = {
       channel: channel,
-      username: SiteSetting.title || "Discourse",
+      username: SiteSetting.chat_integration_slack_username || SiteSetting.title || "Discourse",
```

Comment:
The `||` operator won't work on an empty string:  `"" || "hello"` returns `""`

Reply:
My bad! I don't know how I missed that! I will code a solution using `.present?` and commit the changes soon, sorry for the waste of time

Reply:
Done!